### ISSUE TITLE
Issues importing FileUtils

### ISSUE THREAD

**Issue by Trouler** [shashikumarraja/naughty_string_validator_python#16]
I've been running into issues ever since you updated to start using classes, so for about 3 weeks now. As I was kind of busy atm, I simply decided to use the old version to keep my automated tests running. But I would obviously like to run the latest version.
I'm kinda new to programming, so bare with me.

The problem I'm getting is importing FileUtils, as the import of NSV can't seem to find utils for me.
I've been having some issues lately and decided to reinstall Python to have a clean go at it, but I still end up with the same problem of `cannot import name 'FileUtils' from 'utils'`.

This seems to be a problem of where my utils package is installed, hence the reinstall of Python in hopes of fixing it. I managed to solve the problem by going 1 directory up in the import. As such, I changed line 3 of naughty_string_validator.py to `from .utils import FileUtils`.

This solves my problem locally. I asked a colleague to try this out as well, and he ended up having the exact same problem, with the same solution to fix it.

As I'm quite the noob when it comes to this, I have no idea if the problem is on our end, and how we've both, in that case, have managed to screw up our filepaths for utils.

As I'm running my automated test suite in Codebuild in AWS, I figured I'd try there as well, and I run into the same exact issue on a machine that runs on a clean python:3.7-alpine image.

ImportError while loading conftest '/app/tests/conftest.py'.
tests/conftest.py:10: in <module>
    import connect
connect.py:12: in <module>
    from tests.fixtures import _is_subset
tests/fixtures/__init__.py:13: in <module>
    import naughty_string_validator as nsv
/usr/local/lib/python3.7/site-packages/naughty_string_validator/__init__.py:6: in <module>
    from .naughty_string_validator import Naughty
/usr/local/lib/python3.7/site-packages/naughty_string_validator/naughty_string_validator.py:3: in <module>
    from utils import FileUtils
E   ModuleNotFoundError: No module named 'utils'

**Comment by shashikumarraja** [comment]
@Trouler This is indeed an issue. Can you please provide the version you are using, if it's **0.1.5** you will face the issue.
Can you use the latest version **0.1.6** and confirm if it works for you as I released it recently with fixes.
Meanwhile, I will debug it further in different environments.

**Comment by Trouler** [comment]
@shashikumarraja Unfortunately, I just got home from work. I'm 99% certain I was using 0.1.6 as I'm installing through pip.
I'll try again first thing in the morning tomorrow and update you, just to be sure I wasn't running the wrong one.

**Comment by shashikumarraja** [comment]
I am able to replicate the issue in **python 3.7**. Works fine in python 2.7
```
venv3/lib/python3.7/site-packages/naughty_string_validator/naughty_string_validator.py", line 3, in <module>
    from utils import FileUtils
ModuleNotFoundError: No module named 'utils'
```
I have made the fix by using relative import of utils which is same as the one you mentioned. You are not a noob for sure :)
```
from .utils import FileUtils
```
Pushed and released patch **0.1.7** and tested locally. Working fine now. Also removed version 0.1.6 from pypi.
Need to investigate how the tox tests on environment **py35, py36. py37,py38** kept passing earlier. Maybe the tests need some improvement.

Install the new version and let me know if you face any issues.
You are also welcome to raise pr with suggestions/issue-fixes if you want to contribute to this project.
Thanks a lot for flagging this issue.
Happy coding 👍
